docker-compose.yml

```diff
@@ -33,24 +37,20 @@ services:
       - app-network
 
   db:
-    container_name: db-postgres
+    container_name: db-postgres-${ENVIRONMENT:-development}
     image: postgres:15.3
-    # environment:
-    # POSTGRES_USER: ${POSTGRES_USER}
-    # POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
-    # POSTGRES_DB: db_saline_royale
-    # DATABASE_HOST: postgres
     env_file:
-      - ./backend/development.env
+      - ./backend/${ENVIRONMENT:-development}.env
+
     restart: unless-stopped
     ports:
       - 5432:5432
     networks:
       - back-network
 
   s3:
-    image: zenko/cloudserver
-    container_name: s3
+    image: zenko/cloudserver:8.1-latest-arm64v8
```

Comment:
le arm a vraiment in interet ?

Reply:
Nope, en plus c'est vielle version, j'ai oublié de remettre l'autre